File: src/main/java/com/p2p/lending/service/DeptService.java
package com.p2p.lending.service;

import java.util.List;

import com.p2p.lending.pojo.Dept;

public interface DeptService {
	List<Dept> findall();

	Dept findbyid(Integer did);

	int insert(Dept dept);

	int del(Integer did);

	int upd(Dept dept);
}

File: src/main/java/com/p2p/lending/service/DeptService.kt
package com.p2p.lending.service

import com.p2p.lending.entity.Dept

interface DeptService {
    fun findall(): List<Dept>

    fun findbyid(did: Int?): Dept

    fun insert(dept: Dept): Int

    fun del(did: Int?): Int

    fun upd(dept: Dept): Int
}
